app/views/invite_mailer/invite.html.erb

```diff
@@ -0,0 +1,3 @@
+Hello <%= @github_user.full_name %>,
+
```

Comment:
This sentence doesn't flow very well... 😬

Reply:
The user story #6 spells out what the email should say though. Should we ask if we can change it?